### PR TITLE
chore: added arbitrum support

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Task ID: [OS-841](https://aragonassociation.atlassian.net/browse/OS-841)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

[OS-841]: https://aragonassociation.atlassian.net/browse/OS-841?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by josemarinas — DISMISSED** [review]

File: sdk/src/constants.ts [hunk 1]
```diff
@@ -355,22 +445,6 @@ export const LIVE_CONTRACTS: {
...
-      multisigSetupAddress: activeContractsList.sepolia.MultisigSetup,
-      adminSetupAddress: activeContractsList.sepolia.AdminSetup,
-      addresslistVotingSetupAddress:
-        activeContractsList.sepolia.AddresslistVotingSetup,
-      tokenVotingSetupAddress: activeContractsList.sepolia.TokenVotingSetup,
-      ensRegistryAddress: activeContractsList.sepolia.ENSRegistry,
-    },
   },
 };
 export const ADDITIONAL_NETWORKS: Network[] = [
```
Comment (josemarinas) [inline sdk/src/constants.ts:450]: You are missing it here i believe, there is an override of the getNetwork function in the sdk for the networks that are not supported by ethersv5, so you have to add them here for it to work as well

Comment (Michael-A-Heuer) [reply sdk/src/constants.ts:450]: We were able to deploy with the same settings on osx. I assume Arbitrum is old enough to be supported in the old ethers.

**Comment by Michael-A-Heuer** [comment]
While doing the work, I was thinking that it could it make sense to put the supported networks and etherscan browser and API URLs as a `networks.json` or `networks.ts` in one place.

File: contracts/hardhat.config.ts [hunk 2]
```diff
@@ -70,6 +72,14 @@ export const networks: {[index: string]: NetworkUserConfig} = {
     url: `${apiUrls.baseGoerli}`,
     gasPrice: ethers.utils.parseUnits('0.0000001', 'gwei').toNumber(),
   },
+  abitrum: {
```
Comment (josemarinas) [inline contracts/hardhat.config.ts:75]: There is a typo here

Comment (Michael-A-Heuer) [reply contracts/hardhat.config.ts]: Fixed in af0be22f5a302bdb5ca62aa59691cd7abf8ebf52

File: sdk/src/constants.ts [hunk 3]
```diff
@@ -13,24 +13,28 @@ export const MULTI_FETCH_TIMEOUT = 7 * 1000;
 type GraphqlNetworks =
   | 'mainnet'
   | 'goerli'
+  | 'sepolia'
   | 'polygon'
   | 'mumbai'
-  | 'baseGoerli'
   | 'baseMainnet'
```
Comment (Michael-A-Heuer) [inline sdk/src/constants.ts:19]: Any reason for baseMainnet vs. just base @josemarinas ?

Comment (josemarinas) [reply sdk/src/constants.ts:19]: Its the name satsuma gave to the network

**Review by josemarinas — APPROVED** [review]

**Pull request merged**